packages/opentelemetry-instrumentation-chromadb/tests/test_query.py

```
@@ -1,17 +1,24 @@
 import json
-from os import getcwd
+import tempfile
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)